### ISSUE TITLE
MCM menu not configurable 

### ISSUE THREAD

**Issue by 1Xazam1** [mwilsnd/SkyrimSE-SmoothCam#62]
I Installed this mod recently on the latest anniversary update, and I am not able to interact with any of the Toggle options in the MCM, I wanted to enable the option to show the trajectory of my arrows but due to this I'm not able to

**Comment by mikew** [comment]
Seeing the same, on the latest AE. `%USERPROFILE%\Documents\My Games\Skyrim Special Edition\SKSE\SmoothCam.log` doesn't update when the game launches, I see this in `%USERPROFILE%\Documents\My Games\Skyrim Special Edition\SKSE\skse64.log`:

```
checking plugin SmoothCam.dll
plugin SmoothCam.dll (00000000  00000000) no version data 0 (handle 0)
```

I'm not sure what it means, but the same type of message shows for other SKSE plugins that aren't loading correctly. And I'm also seeing this in `%USERPROFILE%\Documents\My Games\Skyrim Special Edition\SKSE\TrueDirectionalMovement.log`:

```
E:\Repos\CommonLibSSE_AE\src\SKSE\Interfaces.cpp(287): [error] Failed to register messaging listener for SmoothCam
E:\Repos\TrueDirectionalMovement\src\main.cpp(31): [warning] SmoothCamAPI::RegisterInterfaceLoaderCallback reported an error
E:\Repos\CommonLibSSE_AE\src\SKSE\Interfaces.cpp(268): [error] Failed to dispatch message to SmoothCam
E:\Repos\TrueDirectionalMovement\src\main.cpp(68): [warning] SmoothCamAPI::RequestInterface reported an error
```

**Comment by mikew** [comment]
Hm, I just tried re-installing SmoothCam, selected the AE version and the ESL MCM menu, and it worked. Then I reinstalled again, again selecting the AE version but this time the ESP MCM menu, and that too worked. So maybe I totally installed the wrong version at first, but it's something to try. Now it's working fine. The `SmoothCam.log` log isn't updating, but the warning in `skse64.log` is gone, and `TrueDirectionalMovement.log` communicates with SmoothCam as expected.
